core/core-funcs.el

```diff
@@ -119,6 +119,14 @@ Supported properties:
   (re-search-forward anchor-text)
   (beginning-of-line)
   (show-subtree)
+  (setq-local org-emphasis-alist '(("*" bold)
+                                   ("/" italic)
+                                   ("_" underline)
+                                   ("=" org-verbatim verbatim)
+                                   ("~" org-kbd)
+                                   ("+"
+                                    (:strike-through t))))
+
```

Comment:
It won't break your `org-emphasis-alist` for this reason:

`setq-local` only sets the variable value for that buffer. `setq` sets that variable globally.

So, only buffer opened with `spacemacs/open-file` is affected by the above setting to `org-emphasis-alist`.